aviatrix/resource_aviatrix_gateway.go

```
@@ -1474,9 +1474,21 @@ func resourceAviatrixGatewayRead(d *schema.ResourceData, meta interface{}) error
 	}
 
 	if goaviatrix.IsCloudType(gw.CloudType, goaviatrix.AWSRelatedCloudTypes|goaviatrix.AzureArmRelatedCloudTypes) {
-		tags := goaviatrix.KeyValueTags(gw.Tags).IgnoreConfig(ignoreTagsConfig)
-		if err := d.Set("tags", tags); err != nil {
-			log.Printf("[WARN] Error setting tags for (%s): %s", d.Id(), err)
+		tags := &goaviatrix.Tags{
+			ResourceType: "gw",
+			ResourceName: d.Get("gw_name").(string),
+			CloudType:    gw.CloudType,
+		}
+
+		_, err := client.GetTags(tags)
+		if err != nil {
+			log.Printf("[WARN] Failed to get tags for gateway %s: %v", tags.ResourceName, err)
```

Comment to explain why it's ok to ignore this error.